modules/policy/engine/src/main/java/org/eclipse/sw360/antenna/policy/engine/Rule.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.policy.engine;
+
+/**
+ * Interface for a generic rule with the standard fields relevant for all policy rules
+ */
+public interface Rule {
+    /**
+     * @return The business id of the rule defined by the policy responsible
+     */
+    String getId();
+
+    /**
+     * @return The business name of the rule defined by the policy responsible
+     */
+    String getName();
```

Comment:
Does it make sense to have an id *and* a name? What is the purpose of the one vs. the other? Why can't a name serve as the id?

Reply:
To be discussed. Marcels rules have both, an id like A9 and a name like in this case "License with no Threatgroup"